content/learn/accessibility/code-review/index.md

```diff
@@ -28,7 +28,10 @@ Ensure that link text describes the target and purpose and that it can be distin
 
 ## Semantic markup
 
-Ensure that proper semantic HTML elements are used to describe the content of the page or component. For example, <code>&lt;ul&gt;</code> unordered lists are used for lists, <code>&lt;h1&gt;</code>-<code>&lt;h6&gt;</code> are used in order for headings, <code>&lt;nav&gt;</code> is used for navigation.
+Ensure that proper semantic HTML elements are used to describe the content of the page or component. For example:
+- <code>&lt;ul&gt;</code> are used for unordered lists
+- <code>&lt;h1&gt;</code>-<code>&lt;h6&gt;</code> are used in order for headings
+- <code>fieldset</code> and <code>legend</code> are used to [group related sets of controls](https://www.w3.org/WAI/tutorials/forms/grouping/).
```

Comment:
Since we include the angle brackets on the elements in the previous two bullets, shouldn't "fieldset" and "legend" also have angle brackets? ... Also, are these properties for grouping related controls or grouping related sets? I think it's the former. If so, I think that "related sets of controls" should actually be "sets of related controls." But I'm not sure. ... Also, to make this use active voice, I'd start with "Use" and delete "are used."  If we do this, the bullet becomes a complete sentence, so we should also end with a period. (i.e., "Use &lt;fieldset&gt; and &lt;legend&gt; to group ...")